hbs-minifier-plugin.js

```diff
@@ -88,15 +98,26 @@ class HBSMinifierPlugin {
 
     return { name: 'hbs-minifier-plugin', visitor };
   }
-
-  transform(ast) {
-    let plugin = HBSMinifierPlugin.createASTPlugin();
-
-    this.syntax.traverse(ast, plugin.visitor);
-
-    return ast;
-  }
-
 }
 
-module.exports = HBSMinifierPlugin;
+module.exports = function(config) {
+  config = config || {};
+  config = assignDefaultValues(config);
```

Comment:
I am missing the previous default config where (without any configuration) `<pre>` elements and the `{{#no-minify}}` "component" were skipped.

Reply:
I'll add those to the default config.